README.md

```diff
@@ -1,41 +1,22 @@
-Module usage:
-
-     module "rds" {
-        source                = "git::https://github.com/UKHomeOffice/acp-tf-rds?ref=master"
-
-        name                         = "fake"
-        allocated_storage            = "20"
-        cidr_blocks                  = ["${values(var.compute.cidrs)}"]
-        database_name                = "keycloak"
-        database_password            = "password"
-        database_port                = "3306"
-        database_user                = "root"
-        db_parameter_family          = "default.mysql5.6"
-        dns_zone                     = "${var.dns\_zone}"
-        engine_type                  = "MariaDB"
-        engine_version               = "10.1.19"
-        environment                  = "${var.environment}"
-        instance_class               = "db.t2.medium"
-        max_allocated_storage        = 100
-        snapshot_identifier          = "rds:production-2015-06-26-06-05"
-        apply_immediately            = false
-        performance_insights_enabled = true
-
```

Comment:
Why is this being removed?

Reply:
It was meant to be generated by terraform-docs, but wasn't.
I've now amended the formatting of the comment in main.tf and it gets generated as expected.